catalog/NEON/NEON_CHM.jsonnet

```diff
@@ -0,0 +1,169 @@
+local id = 'NEON/CHM';
+local subdir = 'NEON';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'NEON Canopy Height Model',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: ||| 
+	Height of the top of canopy above bare earth (Canopy Height Model (CHM)); 
+	data are mosaicked over the AOP footprint onto a spatially uniform grid 
+	at 1 m spatial resolution. The CHM is generated by creating a continuous 
+	surface of canopy height estimates across the entire spatial domain of 
+	the LiDAR survey, derived from the LiDAR point cloud. The point cloud is 
+	separated into classes representing the ground and vegetation returns. 
+	The ground classified points allow for calculation of a height normalized 
+	point cloud to provide a relative estimate of vegetation elevation. A 
+	surface is then generated using the height normalized vegetation points 
+	to produce the CHM. Any canopy heights less than 2m are set to zero for 
+	data collected with the Optech Gemini LiDAR sensors, and less than 2/3m 
+	for the Riegl and Optech Galaxy sensors. See 
+	[NEON Data Product DP3.30015.001](https://data.neonscience.org/data-products/DP3.30015.001) for more details.
+	Citation: Please use the appropriate citation(s) in your publications; see Terms of Use for more info.
+	Documentation: [NEON DP3.30015.001 Quick Start Guide](https://data.neonscience.org/api/v0/documents/quick-start-guides/NEON.QSG.DP3.30015.001v1?inline=true&fallback=html)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
```

Comment:
You might want to add `canopy` to the list. From my notes on keywords:

`cat canopy_description.md`
- Wiktionary [canopy](https://en.wiktionary.org/wiki/canopy)
- Wikipedia [Canopy (biology)](https://en.wikipedia.org/wiki/Canopy_(biology))
- GCMD [canopy](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/f5078704-ed6b-4064-900c-6a4827ed048b?gtm_keyword=canopy&gtm_scheme=MeasurementName)

Maybe less helpful would be `forest`:

`cat forest_description.md` 
- Wiktionary [forest](https://en.wiktionary.org/wiki/forest)
- Wikipedia [Forest](https://en.wikipedia.org/wiki/Forest)
- Wikipedia [Forestry](https://en.wikipedia.org/wiki/Forestry)
- GCMD
  - [FOREST SCIENCE](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/22ec2f9b-1f1a-469b-bc09-851d58637ff4?gtm_keyword=FOREST%20SCIENCE&gtm_scheme=Earth%20Science)
  - [FORESTS](https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/all/46e4aaa4-349c-4049-a910-035391360010?gtm_keyword=FORESTS&gtm_scheme=Earth%20Science)

Reply:
good suggestion, added those + "vegetation" keyword